istio-control/istio-autoinject/templates/mutatingwebhook.yaml

```diff
@@ -1,16 +1,17 @@
+{{ if not .Values.skipWebhook }}
```

Comment:
can you add this to values.yaml if it is intended to be used? Not sure why someone would want the injector without a webhook, unless there is an existing one pointing to it already?

Reply:
This is used internally by the installer - I don't think it's something that we want users to actually do. 

Installer and operator may install the injector without a webhook, and use some tool to create the secret and inject the webhook manually later.

Or even create a webhook only ( without the autoinjector ) and point it to a deployment on a different cluster. 

Also webhook manipulation requires very high cluster-wide permissions, while upgrading the injector and deployments does not. 

In general I am trying to decouple the (deployment+config+service) from additional cluster wide resources ( like the webhook ).